Formula/docker-ls.rb

```diff
@@ -0,0 +1,36 @@
+class DockerLs < Formula
+  desc "Tools for browsing and manipulating docker registries."
+  homepage "https://github.com/mayflower/docker-ls"
+  url "https://github.com/mayflower/docker-ls.git",
+    :using => :git,
+    :tag => "v0.3.1",
+    :revision => "d80310976c9707e261e57ebfa9acf4e0b1781460"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+
+    mkdir_p buildpath/"src/github.com/mayflower/docker-ls"
+    [
```

Comment:
Any specific reason for avoiding doing [this sort of thing](https://github.com/Homebrew/homebrew-core/blob/a55132a850689800881a70b10dc7c20076f1e5ca/Formula/exercism.rb#L19-L23)?

Reply:
Lack of experience :smirk: I'll revise the PR.